docs/contributing/introduction-to-contributing.md

```diff
@@ -28,13 +28,20 @@ please read the [code of conduct](./code-of-conduct.md) and follow the direction
 
 ## Issues
 
-If you wish to work on an open issue, please comment on the issue with `.take`, and it will be assigned to you. If an issue is not assigned, it is assumed to be available for anyone to work on. Please assign yourself to an issue before working on it to avoid conflicts.
+- If you wish to work on an open issue, please comment on the issue with `.take`, and it will be assigned to you.
 
-If you contribute to the project for the first time, you can search for issues with `good first issue` or `bug` labels in the repository. Please only self-assign a `good first issue` one time.
+> [!TIP]
+> If an issue is not assigned, it is assumed to be available for anyone to work on. Please assign yourself to an issue before working on it to avoid conflicts.
+
+- Issues with the `needs triage` label are unavailable to `.take` until they are triaged and the label is removed. Feel free to check on the issue regularly if you want to work on it.
+- If you contribute to the project for the first time, you can search for issues with `good first issue` or `bug` labels in the repository. Please only self-assign a `good first issue` one time.
+
+> [!NOTE]
+> Please only self-assign a `good first issue` once.
 
```

Comment:
Please change the `[!TIP]` and `[!NOTE]` to what has been asked in the issue, which are `:::info` and `:::note`.

And please give proper indentation to them as what has shown in the copy in the issue.